_posts/2020-07-08-pull-requests-require-context.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: epic
+title: "Pull Requests Need Context"
+date: "2020-07-08"
+author: [steve-hicks]
+categories: [tools, github, team, community, engineering]
+---
+
+Code review is an engineering process that has benefited greatly from a move toward asynchronous communication.
+Long ago, engineering teams would sit around in a room together with code on a projector to review changes. 😱 For
+many teams this would mean batching code reviews or even skipping them altogether. 😱😱
+
+Today, most engineering teams use incredible tools like GitHub or GitLab to review changes through Pull Requests
+(PRs). The greatest advantage of PRs is that the review can happen when it's convenient for the reviewer:
+asynchronously.
+
+<!-- more -->
```

Comment:
Minor suggestion to move it one paragraph down to have a teaser of the "problem".